src/components/AddTask.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from "react";
+import { TaskContext } from "../Context/todoContext.tsx";
+import { TaskContextType, ITask } from '../@types/task';
+import axios from "axios";
+//import styles from './styles.module.ts'
+
+
+
+const NewTask : React.FC<{userToken: string}> = ({userToken}) => {
+    const [formData, setFormData] = React.useState<string>('') ;
+    const { saveTask } = React.useContext(TaskContext) as TaskContextType;
+
+    const addTask = async(task: ITask) =>{
+
+        try{ 
+            await axios.post("http://localhost:8080/tasks", 
+                task,
+                {
+                    headers : {
+                        accesstoken : userToken
+                    }
+                }
+            )
+            console.log(task)
+
+        }catch (error: any) {
+            if (!error.response) {
+                console.log('Network error:', error);
+            } else {
+                console.log('Error response:', error.response);
+            }
+        }
+    }
+    
+    const hendleSubmit = (e: React.FormEvent, formData: any) : void => {
+        e.preventDefault();
+        //{description: formData, status: false, id: 0 } id - va fi schimbat in saveTask
```

Comment:
How can you write this with code / variables? _(without using comments)_

If you can't, just delete it. (+ you could add another task in Trello that contains the details of what needs to be done.)